packages/api-nest/src/graphql/graphql.module.ts

```
@@ -1,22 +1,37 @@
 import { Module } from '@nestjs/common'
+import { ModuleRef } from '@nestjs/core'
 import { ConfigModule, ConfigService } from '@nestjs/config'
 import { GraphQLModule } from '@nestjs/graphql'
 import { ApolloDriver, ApolloDriverConfig } from '@nestjs/apollo'
 import { ApolloServerPluginLandingPageLocalDefault } from '@apollo/server/plugin/landingPage/default'
-import { join, resolve } from 'path'
-import { AuthModule } from '../auth/auth.module'
-import { AuthService } from '../auth/services/auth.service'
+import { Request, Response } from 'express'
```

ESLint

[error] 'Request' is defined but never used.

(@typescript-eslint/no-unused-vars)